### ISSUE TITLE
Saved Places - Go: Falsifies angles and distance to default values 45,0,1500

### ISSUE THREAD

**Issue by DerKarlos** [StrandedKitty/streets-gl#121]
**Describe the bug**
The UI Saved Places stores the actual position exactly.
One can copy the URL and put it in the browser address line. Impressing.
But:
Using Go sets the angles and distances to default values 45.00,0.00,1500.00

**Expected behaviour**
It should be: https://streets.gl/#53.55496,9.99391,5.25,254.00,1869.90
not this one: https://streets.gl/#53.55496,9.99391,45.00,0.00,1500.00

**Map location**
any

**Screenshots**
-/-

**System information**
 - OS: MacOS X
 - Browser with version: Safari, Opera, Crome, Firefox

**Additional context**
Streets GL is a great tool!

**Comment by DerKarlos** [comment]
Meanwhile, I branched StrandedKitty/streets-gl
and by the README, I could set up the development environment,
build, run and debug the Frontend. Great job!
After exploring the modules and sources,
I could see, what's missing to "Go" to the stored state:

The "Go" button calls goToLatLon/setLatLon, which sets the defaults. So I added 
goToState: (lat: number, lon: number, pitch: number, yaw: number, distance: number) => void;
and setState(lat, lon, pitch, yaw, distance)

In SavedPlaceParams and addNewPlace I added pitch, yaw and distance.
I did NOT add them to addNewPlace, that would mean no backward compatibility.
I even removed lat, lon because it is redundant in .link.
In useLayoutEffect, all values are taken from .link now.

Pull Request #150
